nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/connect/client/ui/SharedPackageListingsSettings.java

```diff
@@ -50,16 +83,14 @@ public ListingFilterSetting get(String listName) {
     }
 
     public static SharedPackageListingsSettings instance() {
+        RequestResolver resolver = resolvers.values()
+                                            .stream()
+                                            .filter(RequestResolver::isActive)
+                                            .findFirst()
```

Comment:
Here maybe you could use .map(RequestResolver::resolve).map(this::instance).orElse(null);